appservice/src/intTest/groovy/nu/yona/server/VpnConnectionStatusEventTest.groovy

```diff
@@ -0,0 +1,163 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Stichting Yona Foundation
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v.2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server
+
+import static nu.yona.server.test.CommonAssertions.*
+
+import groovy.json.*
+import nu.yona.server.test.User
+
+class VpnConnectionStatusEventTest extends AbstractAppServiceIntegrationTest
```

Comment:
There are no tests with multiple devices, which may be good to verify

Reply:
Yes, indeed. Added it.